### PR TITLE
Fix c on line beginning#1302

### PR DESCRIPTION
**Pull request by xlaech**
Changed the c Action to encounter for cases where the selection ends at the beginning of a line.
The tests revealed, that s is implemented using c(0,0).

Therefore i check, if this is a real selection before calling the getLeftThroughLineBreaks() function

### CODE REVIEW

File: src/actions/actions.ts [hunk 1]
```diff
@@ -2080,13 +2080,16 @@ export class ChangeOperator extends BaseOperator {
 
     public async run(vimState: VimState, start: Position, end: Position): Promise<VimState> {
         const isEndOfLine = end.character === end.getLineEnd().character;
+        const isBeginning =
```
Comment (johnfn) [inline src/actions/actions.ts]: Position has a function called `isLineBeginning()` - could you use that one?

**Comment by xlaech** [comment]
Sorry i got excited :)

**Comment by xlaech** [comment]
just wait... i need to make sure the tests pass again

**Comment by johnfn** [comment]
Cool, thanks for the PR!

**Pull request merged**